sorc/build_all.sh

```diff
@@ -132,6 +132,20 @@ $Build_gsi && {
 	((err+=$rc))
 }
 
+#------------------------------------
+# build gdas
+#------------------------------------
+$Build_gdas && {
```

Comment:
Please use proper indentation for this block

Reply:
Indent to be consistent with other sections.